app/models/dialog.rb

```diff
@@ -1,8 +1,9 @@
 class Dialog < ApplicationRecord
-  DIALOG_DIR = Rails.root.join("product/dialogs/service_dialogs")
+  DIALOG_DIR_CORE   = 'product/dialogs/service_dialogs'.freeze
+  DIALOG_DIR_PLUGIN = 'content/service_dialogs'.freeze
```

Comment:
I feel like we should just create a content/service_dialogs directory in manageiq proper, so the directories are consistent.